content/docs/user-guide/deploying/kubernetes.md

```diff
@@ -12,6 +21,344 @@ pip install mlem[kubernetes]
 pip install kubernetes docker
 ```
 
+## Description
+
+Deploying to a Kubernetes cluster involves 2 main steps:
+
+1. Build the docker image and upload it to a registry.
+2. Create resources on the Kubernetes cluster -- specifically, a `namespace`, a
+   `deployment` and a `service`.
```

Comment:
Good explanation! Very useful to explain what happens under the hood. Maybe we can add this to other deployments as well? Or to `build docker` user-guide if it's not there (`build` is `serve` packed into Docker Image)? thinking about the other docs here as well

Reply:
I can add take care of this in a separate PR

Reply:
>  Or to build docker user-guide if it's not there (build is serve packed into Docker Image)? thinking about the other docs here as well

This one I did already.

> I can add take care of this in a separate PR

Yes, please do.